src/services/utils.js

```diff
@@ -429,12 +429,21 @@ export const testPowerForPeriods = (
 ) => {
   const rate = values?.rate || values?.tariff
   let valids = 0
+  let inLimit = false
+
   if (rates[rate] === undefined) return true
+
   for (let i = 1; i <= rates[rate]?.num_power_periods; i++) {
     const attr = i === 1 ? 'power' : `power${i}`
-    const inLimit = limit.match('min')
-      ? values[attr] >= rates[rate][limit]?.power
-      : values[attr] <= rates[rate][limit]?.power
+
+    if (limit.match('min')) {
```

Comment:
I don't know if change this method is better than change the `3.0TD` rate specifications in `getRates`. What do you think?

Reply:
The problem is that, in theory, there isn't an upper limit... o_O